src/lib/components/works_for_you/notification.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from "react"
+import { StyleSheet, View, ViewProperties } from "react-native"
+import * as Relay from "react-relay"
+
+import ArtworksGrid from "../artwork_grids/generic_grid"
+import Headline from "../text/headline"
+import SerifText from "../text/serif"
+
+import colors from "../../../data/colors"
+
+interface Props extends ViewProperties {
+  notification: {
+    message: string
+    date: string
+    artists: string
+    status: "READ" | "UNREAD"
```

Comment:
Where does this `status` data come from, as I don’t see it in the Relay fragment?